app/src/main/java/com/github/orkest/domain/FireStoreDatabaseAPI.kt

```diff
@@ -27,14 +27,16 @@ open class FireStoreDatabaseAPI {
 
     companion object{
         val db = Firebase.firestore
+        fun isOnline(context: Context): Boolean {
+            val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+            val network = connectivityManager.activeNetwork ?: return false
+            val networkCapabilities = connectivityManager.getNetworkCapabilities(network) ?: return false
+            return networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)
+        }
```

Comment:
I don't think it's a problem for now, but maybe in the future it would be useful to create a sort of utils folder with static classes in which we could keep functions like these since being connected to the internet is something more general and not specific to the firestore database.

Otherwise, very useful for the rest of the team, thank you for creating this function!